assets/css/oag.css

```diff
@@ -0,0 +1,208 @@
+            background-color: #1A303C;
+            border-color: #091c2d;
+        }
+    }
+
+    .alert-info {
+        border-color: var(--link-color);
+        -o-border-image: linear-gradient(to bottom, var(--link-color) 16px, var(--link-color) 16px, transparent 16px, transparent 48px, var(--link-color) 48px, var(--link-color) 48px) 1 100%;
+        border-image: linear-gradient(to bottom, var(--link-color) 16px, var(--link-color) 16px, transparent 16px, transparent 48px, var(--link-color) 48px, var(--link-color) 48px) 1 100%;
+        color: #676C72;
+
+        &::before {
+            color: var(--link-color);
+        }
+
+        & > :last-child {
+           padding-bottom: 0px;
+        }
+
+    }
+    .flex-container {
+        display: flex;              
+        flex-direction: column;    
+        justify-content: space-between; 
+        height: 100%;              
+        position: relative;
+        h3 {
+            width: fit-content;
+        }
+        .alert.alert-info {
+            margin-bottom: auto;       
+        }        
+    }   
+   
+    .btn-link {
+        color: var(--link-color);
+        background: linear-gradient(to right, var(--hover-color) 0%, var(--hover-color) 100%);
+        background-size: 0% 100%;
+        background-repeat: no-repeat;
+        transition: background-size 0.3s ease-in-out;
+        position: relative;
+        overflow: hidden;
+    
+        &:hover {
+            background-size: 100% 100%; 
+            color: var(--primary-color);
+            text-decoration: none;
+        }
+        &:not(:hover) {
+            background-size: 0% 100%;
+            transition: none;
+        }
+    }
+        
+    .label-primary {
+        background: #E3E0D3;
+        border-color: #908F84;
+        color: #4B5B62;
+    }
+
+    .text-primary {
+        color: var(--primary-color);
+    }
+
+    .well {
+        border: 2px solid #908F84;
+    }
+
+    .home {
+        .report {
+            section {
+                background-color: white;
+            }
+        }
+    }   
+
+    .panel-default {
+        border: 1px solid transparent;
+        background-color: #F4F4F2;
+    }
+    blockquote footer {
+        font-size: 100%;
+        line-height: 1.4375;
+        color: #262d35;
+        cite span {
+            font-weight: 700;
+        }
+    }
+    .reports {
+        font-size: 20px;
+        line-height: 1.2;
+    
+        hgroup {
+            display: flex;
+            align-items: center;
+            gap: 15px;
+    
+            h3, p {
+                margin: 0;
+            }
+
+            h2 + p {
+                font-weight: 700;
+            }
+    
+            p {
+                font-size: 14px;
+                font-weight: 700;
+            }
+        }
+    }
+
+    hgroup {
+        h2 + p {
+            font-weight: 700;
+        }
+    }
+
+    .wb-eqht-grd > [class*="col-"] > section .pull-right {
+        margin-left: auto;
+    }
```

Comment:
We might already have something for margin-left: auto.

because we do have "col-md-auto" that exist + recently we do have a full set of spacing utilities: https://wet-boew.github.io/GCWeb/common/spacing/spacing-en.html